app/controllers/application_controller.rb

```diff
@@ -12,7 +12,7 @@ class ApplicationController < ActionController::Base
   def authenticate_for_staging
     if ENV['USE_HTTP_BASIC'] && !request.format.to_s.match('json') && !params[:print] && !request.path.include?('api') && !request.path.include?('pdf')
       authenticate_or_request_with_http_basic do |username, password|
-        username == "oralhistory" && password == "oralhistory"
+        username == ENV['STAGING_HTTP_BASIC_USERNAME'] && password == ENV['STAGING_HTTP_BASIC_PASSWORD']
```

Comment:
Brakeman Audit:
Confidence: High
Category: Basic Auth
Check: BasicAuth
Message: Basic authentication password stored in source code
Code: (password == "oralhistory")
File: app/controllers/application_controller.rb
Line: 15

Reply:
We will have to let John know that he will need to add env variables to staging and test